frontend/src/pages/modelServing/screens/projects/kServeModal/ManageKServeModal.tsx

```diff
@@ -412,6 +412,9 @@ const ManageKServeModal: React.FC<ManageKServeModalProps> = ({
         {!hideForm && (
           <FormSection title="Source model location" id="model-location">
             <ConnectionSection
+              existingUriOption={
+                editInfo?.inferenceServiceEditInfo?.spec.predictor.model?.storageUri
+              }
```

Comment:
Ah - @manaswinidas for https://issues.redhat.com/browse/RHOAIENG-19258 this is where we could pass our URI when prefilling from model registry if it is a RH registry URI (starts with `oci://registry.redhat.io/`). We probably want that logic encapsulated somewhere MR specific but we can maybe lift out an optional `existingUriOption` to be passed into ManageKServeModal and use it if present instead of this one from `editInfo`.